samples/powerpoint/shapes/get-set-shapes.yaml

```
@@ -0,0 +1,223 @@
+order: 2
+id: powerpoint-shapes-get-set-shapes
+name: 'Get, set, load, and save shapes'
+description: Get and set one or more selected shapes. Load and save one or more shapes.
+host: POWERPOINT
+api_set:
+    PowerPointApi: '1.5'
+script:
+    content: |
+        $("#getSelectedShapes").click(() => tryCatch(getSelectedShapes));
+        $("#setSelectedShapes").click(() => tryCatch(setSelectedShapes));
+        $("#changeFill").click(() => tryCatch(changeFill));
+        $("#saveShapeSelection").click(() => tryCatch(saveShapeSelection));
+        $("#loadShapeSelection").click(() => tryCatch(loadShapeSelection));
+        $("#createShapes").click(() => tryCatch(createShapes));
+        $("#arrangeSelected").click(() => tryCatch(arrangeSelected));
+
+        async function getSelectedShapes() {
+          await PowerPoint.run(async (context) => {
```

Do include descriptions for the various functions
```suggestion
        async function getSelectedShapes() {
          // Insert description here...
          await PowerPoint.run(async (context) => {
```

---

added these in.